### ISSUE TITLE
[a11y] avoid TYPE_WINDOW_STATE_CHANGED for announcing route names

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#86575]
See https://github.com/flutter/engine/blob/eaf70ca01903d25f2cd7aad2f0f9ce754e1e0e62/shell/platform/android/io/flutter/view/AccessibilityBridge.java#L1787-L1804.

As of Android 11 newer versions of TalkBack may not announce the message provided to the TYPE_WINDOW_STATE_CHANGED event. ~I believe we still need to fire the event to notify that the UI has changed in a significant way, but we shouldn't attach the route name to the event.~ _Edit: We should never manually dispatch this event._ Instead, the recommended way to announce route changes is to change the window or fragment/view title using one of these APIs:

* [Activity#getWindow()#setTitle(CharSequence)](https://developer.android.com/reference/android/view/Window#setTitle(java.lang.CharSequence))
* [ViewCompat.setAccessibilityPaneTitle(View, CharSequence)](https://developer.android.com/reference/androidx/core/view/ViewCompat#setAccessibilityPaneTitle(android.view.View,%20java.lang.CharSequence))

**Comment by goderbauer** [comment]
Related: https://github.com/flutter/flutter/issues/86577

**Comment by chunhtai** [comment]
is it currently broken for android 11? or is this a `to be deprecated` thing

**Comment by goderbauer** [comment]
On my Android 11 device with TalkBack 9 it looks like these announcements are not happening anymore...

**Comment by chunhtai** [comment]
@goderbauer I can't reproduce this with my pixel5 android 11 talkback9

This is the code i used

<details>

```dart
import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Material App',
      theme: ThemeData.dark(),
      home: Home(),
    );
  }
}

class Home extends StatefulWidget {
  @override
  State<Home> createState() => _HomeState();
}

class _HomeState extends State<Home> {
  double _currentSliderValue = 20;

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('Material App Bar'),
      ),
      body: Center(
        child: RaisedButton(
          child: Text('press'),
          onPressed: () {
            Navigator.of(context).push(MaterialPageRoute(builder: (_) => SecondPage()));
          },
        ),
      ),
    );
  }
}

class SecondPage extends StatefulWidget {
  @override
  State<SecondPage> createState() => _SecondPageState();
}

class _SecondPageState extends State<SecondPage> {
  double _currentSliderValue = 20;

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('Material App Bar'),
      ),
      body: Center(
        child: Semantics(
          namesRoute: true,
          scopesRoute: true,
          explicitChildNodes: true,
          label: 'My new name route',
          child: Text('TA LA !'),
        ),
      ),
    );
  }
}
```

</details>

When i press the button I still heard the `My new name route` feedback

**Comment by goderbauer** [comment]
Interesting. Either way, we have to stop dispatching that event, though, as it may stop working in the future.
